packages/transformer/src/IModelTransformer.ts

```diff
@@ -2890,19 +2959,35 @@ export class IModelTransformer extends IModelExportHandler {
         )
           continue;
         await this.processDeletedOp(
-          change,
+          change.ECInstanceId,
+          change.$meta?.classFullName ?? "",
```

Comment:
- What is means when empty string is passed for classFullName? Will it still produce expected results?   
- It could be a good idea to split `processDeletedOp()` into `processRelationshipDeleteOp()` and `processElementDeleteOpt()`, as it seems that separate sets of variables are needed to process these 2, e.g. className, sourceIdOfRelationshipInSource and targetIdOfRelationshipInSource  are not needed when processing element delete operation.

Reply:
I explored your first bullet point and seems like the answer is no. We would fail if we called 'queryProvenanceForRelationship' with "" and we would also fail when wewent to query the deletedSourceRelationshipData in order to delete it in the target. For now I've decided to log an error if this scenario occurs warnning that the relationship delete may not propagate. Im not sure its worth throwing the entire transformer run away for, but maybe it is. Right now I've decided to not do that.

Reply:
Refactored processDeletedOp as well